README.md

```diff
@@ -0,0 +1,381 @@
+        "credit": 100
+      }
+    },
+    {
+      "type": "TimeBasedRate",
+      "id": 3,
+      "currency": "EUR",
+      "interval": {
+        "timeAmount": 90,
+        "timeUnit": "Minutes"
+      },
+      "pricePerInterval": {
+        "credit": 100
+      }
+    }
+  ],
+  "slots": [
+    {
+      "rate": 2,
+      "start": {
+        "timeAmount": 0,
+        "timeUnit": "MINUTES"
+      },
+      "end": {
+        "timeAmount": 2,
+        "timeUnit": "HOURS"
+      }
+    },
+    {
+      "rate": 3,
+      "start": {
+        "timeAmount": 2,
+        "timeUnit": "HOURS"
+      }
+    }
+  ]
+}
+```
+
+This tariff has two slots with an associated rate. The first slot begins at the start of the trip and covers the first
+two hours. It does not matter if the bike is rented for ten minutes or two hours. During this time, one euro is charged.
+The second slot starts at two hours and is not limited. The TimeBasedRate belongs to this slot. If you now ride for
+three hours, one euro from the second slot is added to the one euro from the first slot, because the interval was
+started once. If you travel for five hours, three euros from the second slot are added to the one euro from the first
+slot, because the interval was started three times.
+
+It is also possible to define a **billing interval**. This causes that after this interval the slots are calculated from
+the beginning.
+
+###### Example 2
+
+```JSON
+{
+  "type": "SlotBasedTariff",
+  "id": 1,
+  "currency": "EUR",
+  "billingInterval": {
+    "timeAmount": 5,
+    "timeUnit": "HOURS"
+  },
+  "rates": [
+    {
+      "type": "FixedRate",
+      "id": 2,
+      "currency": "EUR",
+      "price": {
+        "credit": 100
+      }
+    },
+    {
+      "type": "TimeBasedRate",
+      "id": 3,
+      "currency": "EUR",
+      "interval": {
+        "timeAmount": 90,
+        "timeUnit": "MINUTES"
+      },
+      "pricePerInterval": {
+        "credit": 100
+      }
+    }
+  ],
+  "slots": [
+    {
+      "rate": 2,
+      "start": {
+        "timeAmount": 0,
+        "timeUnit": "MINUTES"
+      },
+      "end": {
+        "timeAmount": 2,
+        "timeUnit": "HOURS"
+      }
+    },
+    {
+      "rate": 3,
+      "start": {
+        "timeAmount": 2,
+        "timeUnit": "HOURS"
+      }
+    }
+  ]
+}
+```
+
+Up to a travel time of five hours, this rate is calculated identically to [Example 1](#example-1). After 5 hours, the
+first slot is started again and added up. This means that for a trip of eight hours, one euro from the first slot and
+one euro from the second slot are added to the four euros from the first billing interval. This is repeated with each
+subsequent run of a billing interval.
+
+
+### TimeBasedTariff
+
+A TimeBasedTariff is also composed of individual slots. However, the calculation does not start with the first slot, but
+with the one corresponding to the start time. All slots have a start time and an end time. These times always refer to a
```

Comment:
Let's rephrase this and maybe say something like 'first point in time intersecting'. @jonasvogel , do you have an idea? I